zio-schema-zio-test/shared/src/main/scala/zio/schema/DeriveGen.scala

```diff
@@ -0,0 +1,1022 @@
+package zio.schema
+
+import zio.random.Random
+import zio.schema.Schema._
+import zio.test._
+
+object DeriveGen {
+
+  def gen[A](implicit schema: Schema[A]): Gen[Sized with Random, A] =
+    schema match {
+      case GenericRecord(structure)             => StructureGen.listMap(structure)(gen(_))
+      case Sequence(schemaA, fromChunk, _)      => Gen.chunkOfBounded(10, 100)(gen(schemaA)).map(fromChunk)
```

Comment:
```suggestion
      case Sequence(schemaA, fromChunk, _)      => Gen.chunkOf(gen(schemaA)).map(fromChunk)
```

Reply:
These values seem somewhat arbitrary. I would suggest using `Gen.chunkOf` instead of `Gen.chunkOfBounded` so that the test framework can automatically generate a collection of the appropriate size using the `Sized` service.